mpl_sphinx_theme/__init__.py

```diff
@@ -33,9 +33,16 @@ def get_html_theme_path():
 
 def setup_html_page_context(app, pagename, templatename, context, doctree):
     """Add a mpl_path template function."""
+    # Pick link setting based on release mode.
```

Comment:
Can this get more explanation?  I'm not sure what is going on here and why...

Reply:
Sure, I've expanded on a the comments.